File: 88. Merge Sorted Array.java
class Solution {
    public void merge(int[] nums1, int m, int[] nums2, int n) {
        int j = n-1;
        int i = m-1;
        while(i+j+1>=0) {
        	if(j==-1) {
        		break;
        	}
        	else if (i==-1) {
        		nums1[j+i+1] = nums2[j];
        		j--;
        	}
        	else {
        		int n1=nums1[i], n2=nums2[j];
        		int num = Math.max(n1, n2);
        		nums1[j+i+1] = num;
        		if(n1==num) i--;
        		else j--;
        	}
        }
    }
}

----------------------------------------------------------------
class Solution {
    public void merge(int[] nums1, int m, int[] nums2, int n) {
        for(int i=m;i<nums1.length;i++){
            nums1[i]=nums2[i-m];
        }
        Arrays.sort(nums1);
            
        }
        
    }

File: 88. Merge Sorted Array.py
class Solution:
    def merge(self, nums1: List[int], m: int, nums2: List[int], n: int) -> None:
        """
        Do not return anything, modify nums1 in-place instead.
        """
        
        for num in nums2:
            pos=bisect.bisect_left(nums1[:m],num)
            nums1.insert(pos,num)
            nums1.pop()
            m+=1
        
----------
class Solution:
    def merge(self, nums1: List[int], m: int, nums2: List[int], n: int) -> None:
        """
        Do not return anything, modify nums1 in-place instead.
        """
        for i in range(n):
            nums1[i+m]=nums2[i]
        nums1.sort()
        
        
